src/test/java/org/alfresco/repo/event2/DeleteRepoEventIT.java

```diff
@@ -44,6 +46,14 @@ public void deleteContent()
         NodeRef nodeRef = createNode(ContentModel.TYPE_CONTENT);
```

Comment:
Please add "cm:title" property when creating the node so we get a non-empty properties map.